utils/docker/Dockerfile.ubuntu.20.04

```
@@ -28,13 +28,10 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
             libssl-dev libtool-bin libyaml-dev                          \
             locales make meson nasm ninja-build pandoc patch            \
             pylint python-dev python3-dev scons sg3-utils uuid-dev      \
-            yasm valgrind libhwloc-dev man
+            yasm valgrind libhwloc-dev man fuse3
```

fuse is still being built by this PR in the Travis build.  You probably want `fuse3-dev` here.
```suggestion
            yasm valgrind libhwloc-dev man fuse3-dev
```

Since the older Ubuntu was the last distro that needed scons to build fuse, should this PR be removing that as a build-able component entirely, to ensure the build fails if the `fuse3-dev{,el}` prerequisite is not met?